### ISSUE TITLE
CLDRCommand, plural rules gone wild

### ISSUE THREAD

**Issue by My6UoT9** [yiisoft/yii#4094]
I wanted to update the i18n data, as  there was an update on 2016-09-21.

The build command passed without error. But checking the generated files,  I found the plural rules to be awkward:

```
 'pluralRules' => 
  array (
    0 => 'i = 1&&v = 0 @integer 1',
    1 => ' @integer 0, 2~16, 100, 1000, 10000, 100000, 1000000, … @decimal 0.0~1.5, 10.0, 100.0, 1000.0, 10000.0, 100000.0, 1000000.0, …',
    2 => 'true',
  ),
```

previously it was like this:

```
 'pluralRules' => 
  array (
    0 => 'n==1',
    1 => 'true',
  ),
```

Having now checked the release Notes, it seems quite a few things have changed.
http://cldr.unicode.org/index/downloads/cldr-30

**Comment by cebe** [comment]
for which language was that? I remember that CLDR changed the format in which pulral rules are defined so yii command may not be compatible with the new rules anymore.

**Comment by My6UoT9** [comment]
The example is from german language file.
E.g. "de.php"

**Comment by My6UoT9** [comment]
sry for closing, new to github - pressed wrong button

**Comment by samdark** [comment]
It's a known issue. CLDR format changed since the time we wrote the parse so it doesn't work with new versions anymore.

**Comment by samdark** [comment]
Won't be fixed in 1.1.
